Packages/com.unity.inputsystem/InputSystem/Utilities/ArrayHelpers.cs

```diff
@@ -754,5 +754,14 @@ public static void SwapElements<TValue>(this NativeArray<TValue> array, int inde
             array[index1] = array[index2];
             array[index2] = temp;
         }
+
+        public static T[] ToArray<T>(this Array array)
```

Comment:
Can you not just cast the array?
`public static T[] ToArray<T>(this Array array) => (T[])array;`

Reply:
Turns out we can in this case. Thought I was using this for some System.Arrays returned from the reflection API but nope, not the case.